src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/domain/Vilkårperiode.kt

```diff
@@ -29,6 +38,11 @@ data class Vilkårperiode(
     }
 }
 
+enum class KildeVilkårsperiode {
+    MANUELL,
+    SYSTEM,
```

Comment:
Her må vi kanskje si hvilket system vi henter fra senere?

Reply:
Tror ikke vi trenger det? Iaf ikke for målgruppe, der skal vi vel kun hente perioder fra det system som gir oss informasjonen, eks AAP skal hentes fra AAP, EF fra EF etc